docs/publish-integrations/common-publish-integrations/get-approval-via-email.md

```diff
@@ -27,6 +27,14 @@ Upon restarting or initiating the step, it resets all answers to `Waiting`. User
 
 :::
 
+:::tip Get Approval via Email
+
+Get Approval via Email step can be used for different purposes. Since this step takes 2 different parameters, one Required and one Optional, the usage varies. 
```

Comment:
```suggestion
The Get Approval via Email step can be used for different purposes. Since this step takes two different parameters, one Required and one Optional, the usage varies. 
```